LICENSE.txt

```diff
@@ -1,28 +1,4 @@
-Copyright (c) 2013 - 2020, Inria
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-1. Redistributions of source code must retain the above copyright
-   notice, this list of conditions and the following disclaimer.
-2. Redistributions in binary form must reproduce the above copyright
-   notice, this list of conditions and the following disclaimer in the
-   documentation and/or other materials provided with the distribution.
-3. All advertising materials mentioning features or use of this software
-   must display the following acknowledgement:
-   This product includes software developed by the Inria.
-4. Neither the name of the Inria nor the
-   names of its contributors may be used to endorse or promote products
-   derived from this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY Inria ''AS IS'' AND ANY
-EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL Inria BE LIABLE FOR ANY
-DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
-(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
-ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Copyright (c) 2013, IHU Liryc, Université de Bordeaux and Inria. All rights reserved.
```

Comment:
You can't replace a copyright that states: "Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer" with a new one. The whole point of a copyright like this to prevent someone from stealing the code and claiming it for themselves, and this applies to us. We must keep the above copyright and find out how you are supposed to append to it when you legally modify copyrighted code.

Reply:
i copied our previous license on music 2.4.0 on https://github.com/Inria-Asclepios/medInria-public/blob/master/LICENSE.txt
@msermesant do you know how we need to do?

Reply:
I know this was done before but it suddenly occurred to me that we can't do that while reading the PR.

Reply:
we should keep the medInria license on the medInria code and the MUSIC license on the MUSIC-only code.

Reply:
but this license is the one displayed on the application: homepage -> About -> License :/

Reply:
we should be able to customise the license file so that we cite medInria to respect the BSD4 license and then display the MUSIC license

Reply:
I do not understand how technically i need to do that. Do we append this medInria license with our own in this file? Or we create a LICENSE_MUSIC.txt file for instance, that we display in the application? Or we display both files in the application?

Reply:
we create a LICENSE_MUSIC.txt file that we display in the application.